baybe/utils/dataframe.py

```diff
@@ -601,3 +601,63 @@ def get_transform_objects(
         )
 
     return [p for p in objects if p.name in df]
+
+
+def filter_df(
```

Comment:
Thanks for making this an extra function and providing an example. This really made the code cleaner 👍